### ISSUE TITLE
S3 Code is Proprietary 

### ISSUE THREAD

**Issue by russellpierce** [zapier/parquetr#2]
How we hook in and out of S3 is a chunk of code that is Zapier proprietary.  We need to isolate that code, clean it up, and release it open source.  Until that is done, the package is only useful as a starting point for other folk.

**Comment by nickthetait** [comment]
From a zapier-external person looking to contribute, this issue seems like a pretty big roadblock.

**Comment by russellpierce** [comment]
I agree! The proprietary code can be swapped out if we can locate a decent S3 package (e.g. supports multipart uploads without a hassle). Do you have any ideas? My current default 'idea' is writing up a package that uses reticulate and boto3, getting it up on CRAN and then returning here.

**Comment by russellpierce** [comment]
For reference what needs to be swapped out is s3_accessibility_layer and related code.
